core/src/main/kotlin/com/malinskiy/marathon/extension/StringExtensions.kt

```diff
@@ -24,3 +24,5 @@ fun String.escape(): String {
 }
 
 val escapeRegex = "[^a-zA-Z0-9\\.\\#]".toRegex()
+
+fun String.addFileNumberForVideo(fileNumber: String) = "${this.split(".mp4")[0]}$fileNumber.mp4"
```

Comment:
This looks like very specific thing for stringextensions. I suggest putting this somewhere closer to the video file creation